### PR TITLE
2910 Synthesis

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Adds research synthesis markdown & updates readme accordingly

Both can be demoed in the branch https://github.com/raft-tech/TANF-app/tree/2910synthesis/docs/User-Experience/Research-Syntheses

Note that the readme link will not work until the branch is merged.

### CODE REVIEW

**Comment by reitermb** [comment]
Corrected my merging problem & opened up a new branch!

cc @ADPennington

File: docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
...ew)</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3095">#3095</a></td><td>4.0 / P2</td></tr><tr><td>Rearrange Data Files filters and implement multiselect control for STT filtering</td><td>Adds the multiselect control for STT filtering on the Data Files page and rearanges filters into a more intuitive order.</td><td>Data Files</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3097">#3097</a></td><td>4.0 / P2</td></tr><tr><td>[Spike] Investigate YYYYMMDD value filtering for data files</td><td>Higher lift ideal solution following on from #3076. Investigates how we might add the ability to filter the DAC Data Files page to those submitted on a specific date or within a specific date range. </td><td>Data Files</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3077">#3077</a></td><td>Beyond 4.0 / P2</td></tr><tr><td>List of Cat 1,4 rejected case numbers</td><td>Provides a method of filtering current django outputs to produce a list of case numbers on rejected data files.</td><td>Data File Summaries (Specific Summary view)</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3096">#3096</a></td><td>Beyond 4.0 / P2</td></tr></tbody></table>
```
Comment (ADPennington) [inline docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md]: @reitermb please check the bracket around the row referencing #3092

File: docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* [Background](#background)
+* [What we did & who we talked to](#what-we-did--who-we-talked-to)
+* [What we learned](#what-we-learned)
+* [Next steps](#next-steps)
+
+***
+
+## Background
+
+The Django Admin Console (DAC) is an internal tool tailored for our OFA System Admin and DIGIT Team users. It allows for access to the Postgres database & system logs, managing user permissions, and has been increasingly adopted to provide the DIGIT team with quick insights into data file errors and STT submissions.&#x20;
```
Comment (ADPennington) [inline docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md:14]: i think its important to clarify here that the permissions for each group are different. I wouldn't want to misrepresent privileged access. digit team's access to DAC is limited/not privileged. and actually needs to be locked down further (follow-on ticket)

File: docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md (hunk 1 above)
Comment (ADPennington) [inline docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md]: re: #3095, my impression is that the summaries are in json format.

Comment (ADPennington) [reply docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md]: re: #3097 -- confirmed with Yun that she does not need multiselect on STTs on data files page and also wants to combine year and quarter into one fiscal period filter (as shown in search indexes pages) with multiselect option.

Comment (ADPennington) [reply docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md]: re: #3096 -- the explanation needs clarity. OFA groups want  to be able to see which case/months are associated with cat 1 and cat 4 errors. please see legacy system transmission emails (attachments) for reference.

**Review by ADPennington — CHANGES_REQUESTED** [review]
@reitermb please see comments regarding changes requested.

**Comment by reitermb** [comment]
> @reitermb please see comments regarding changes requested.

Everything should be updated now! Also fixed one last formatting issue that was making some of the tables look funky.

File: docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+***
+
+## What we did & who we talked to
+
+We ran two workshops with the DIGIT team; which overlaps with our two OFA System Admins. While those teams overlap, note that the permissions for each user group differ. OFA System Admins have priveleged access to DAC while DIGIT team users have non-priveleged access. Our product manager and two developers were also in attendance for alignment and work estimation purposes. These workshops facilitated conversation around DAC enhancement requests & pain points provided by OFA in tickets [#2930](https://github.com/raft-tech/TANF-app/issues/2930), [#1662](https://github.com/raft-tech/TANF-app/issues/1662), [#960](https://github.com/raft-tech/TANF-app/issues/960), and [#2910](https://github.com/raft-tech/TANF-app/issues/2910) in order to achieve:
```
Comment (ADPennington) [inline docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md]:
```suggestion
We ran two workshops with the DIGIT team; which overlaps with our two OFA System Admins. While those teams overlap, note that the permissions for each user group differ. OFA System Admins have privileged access to DAC while DIGIT team users have non-privileged access. Our product manager and two developers were also in attendance for alignment and work estimation purposes. These workshops facilitated conversation around DAC enhancement requests & pain points provided by OFA in tickets [#2930](https://github.com/raft-tech/TANF-app/issues/2930), [#1662](https://github.com/raft-tech/TANF-app/issues/1662), [#960](https://github.com/raft-tech/TANF-app/issues/960), and [#2910](https://github.com/raft-tech/TANF-app/issues/2910) in order to achieve:
```

**Review by ADPennington — CHANGES_REQUESTED** [review]
- minor change requested
- recommendation to refine any attached tickets that do not currently include visuals of the behavior needing the enhancement (not needed for PR approval, but for reference to those who review this synthesis in the future) 

@victoriaatraft @reitermb @robgendron

**Review by ADPennington — APPROVED** [review]
Lgtm 👍🏾 cc: @ttran-hub @lfrohlich

**Comment by victoriaatraft** [comment]
> * recommendation to refine any attached tickets that do not currently include visuals of the behavior needing the enhancement (not needed for PR approval, but for reference to those who review this synthesis in the future)

Also noted @ADPennington :)

**Pull request merged**